lib/logger/lib/logger/erlang_handler.ex

```diff
@@ -65,7 +105,7 @@ defmodule Logger.ErlangHandler do
 
     metadata =
       case map do
-        %{file: file, line: line} -> [file: List.to_string(file), line: line] ++ metadata
+        %{file: file, line: line} -> [file: to_string(file), line: line] ++ metadata
```

Comment:
Is the change of the `file` type a backwards incompatible change?

Reply:
It is compatible. Just right now there are 2 possibilities:

- when log comes from the Erlang macros - the `file` will be charlist
- when log comes from the Elixir macros - the `file` will be binary

That is why there is `Kernel.to_string/1` used instead of `List.to_string/1`.